src/index.tsx

```diff
@@ -328,17 +338,7 @@ export class App extends React.Component<any, AppState> {
   };
 
   private onKeyDown = (event: KeyboardEvent) => {
-    if (event.key === KEYS.ESCAPE && !this.state.draggingElement) {
-      elements = clearSelection(elements);
-      this.setState({});
-      this.setState({ elementType: "selection" });
-      if (window.document.activeElement instanceof HTMLElement) {
-        window.document.activeElement.blur();
-      }
```

Comment:
I noticed you didn't keep this for the `actionFinalize`. This has the effect the when you select a tool and hit escape, the focus is retained on the tool, not on the `selection` tool that's not selected.

It also doesn't unfocus inputs (which arguably goes against default browser behavior, but I thought it works well with our app).

Reply:
Added it back.